src/services/ksSetting.ts

```diff
@@ -0,0 +1,61 @@
+import { ChainId } from '@kyberswap/ks-sdk-core'
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+
+import { KS_SETTING_API } from 'constants/env'
+
+export type KyberswapConfigurationResponse = {
+  data: {
+    prochart: boolean
+    rpc: string
+    'block-subgraph': string
+    'classic-subgraph': string
+    'elastic-subgraph': string
+  }
+}
+
+type KyberswapGlobalConfigurationResponse = {
+  data: {
+    banner: {
+      id: string
+      name: string
+      start: string
+      end: string
+      link: string
+      img: {
```

Comment:
img { 
mobile,
tablet,
desktop,
}

FE can check isInmodal/media and use the correct one

Reply:
Tin chuan chua anh

Reply:
How about some special case that might required some special screenwidth

Reply:
Maybe it is not neccessary. I will update by your suggestion

Reply:
Updated